react/features/video-menu/components/web/GrantModeratorButton.tsx

```diff
@@ -19,7 +19,7 @@ class GrantModeratorButton extends AbstractGrantModeratorButton {
     constructor(props: IProps) {
         super(props);
 
-        this._handleClick = this._handleClick.bind(this);
+        this._onClick = this._onClick.bind(this);
```

Comment:
Because _onClick already calls _handleClick in the AbstractButton class. And the prevent notification system wouldn't have worked otherwise - it would've meant implementing it in each _handleClick. I found it weird that it was done this way

Reply:
Right. See above, let's refactor this not to inherit from a toolbar button please.